app/Models/Image.php

```diff
@@ -191,4 +193,23 @@ public function ensureDirectoryExists()
             mkdir($destination, 0777, true);
         }
     }
+
+    /**
+     * Deletes any generated files if they exist
+     * @param bool $includeOriginal Deletes
+     */
+    public function clearExisting($includeOriginal = false) {
```

Comment:
New functions being added to the PFM codebase should have static type declarations whenever possible. For this function, that means the signature should look like the following:

`public function clearExisting(bool $includeOriginal = false) {`